doc/src/guide/connection_draining.asciidoc

```diff
@@ -67,3 +67,17 @@ after DrainTimeout ->
 end,
 ok = ranch:stop_listener(Ref).
 ----
+
+If you want to drain listeners automatically as part of your application shutdown routine,
+just put it in the `prep_stop/1` function of your application module and export it.
+
+.Draining listeners automatically on application shutdown
+
+[source,erlang]
+----
+prep_stop(State) ->
```

Comment:
Make it a example snippet including the start and stop callbacks.

Reply:
okay

Reply:
better?